### PR TITLE
Translations german: Reset Password

### PR DESCRIPTION
**Pull request by ksuess**
just a little change to prevent confusion about what is sent: link not password itself.

### CODE REVIEW

File: locales/de/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -1137,7 +1137,7 @@ msgstr "An den Anfang verschieben"
 
 #: components/theme/PasswordReset/PasswordReset
 msgid "My email address is"
-msgstr "Meine E-Mail-Adresse lautet"
+msgstr "Mein Benutzername / meine E-Mail-Adresse lautet"
```
Comment (tisto) [inline locales/de/LC_MESSAGES/volto.po:1140]: @ksuess wouldn't that be upper-case "Meine E-Mail-Adresse"?

**Review by tisto — CHANGES_REQUESTED** [review]
LGTM. Apart from the one minor issue that I mentioned. I am wondering if it would be possible to show a message dependent if email-usernames are enabled or not.

**Comment by ksuess** [comment]
> LGTM. Apart from the one minor issue that I mentioned. I am wondering if it would be possible to show a message dependent if email-usernames are enabled or not.

Maybe I'm wrong, but I think the info about email or username authentication is not available for anonymous.

**Comment by ksuess** [comment]
Test failure seems to be unrelated:

```
Run actions/upload-artifact@v1
Error: Path does not exist /home/runner/work/volto/volto/cypress/videos
Error: Exit code 1 returned from process: file name '/home/runner/runners/2.278.0/bin/Runner.PluginHost', arguments 'action "GitHub.Runner.Plugins.Artifact.PublishArtifact, Runner.Plugins"'.

```

**Review by sneridagh — APPROVED** [review]

**Pull request merged**